module-apps/apps-common/widgets/ProgressTimer.cpp

```diff
@@ -55,6 +55,7 @@ namespace app
     auto ProgressTimer::onTimerTimeout(sys::Timer &task) -> bool
     {
         ++elapsed;
+        update();
```

Comment:
Doesn't it change the behavior of meditation timer?

Reply:
I think that logically this is correct - last tick should also cause to update progress. In this case to `-0:00`. I checked the Meditation app on simulator and everything seems to work correctly.